web/src/main/webapp/WEB-INF/jsp/team/match/detail.jsp

```diff
@@ -0,0 +1,218 @@
+                                                <th>Firstname</th>
+                                                <th>Lastname</th>
+                                            </tr>
+                                        </thead>
+                                        <tbody>
+                                            <c:forEach items="${matchDetail.homeRoster}" var="player">
+                                                <tr>
+                                                    <td>${player.firstname}</td>
+                                                    <td>${player.surname}</td>
+                                                </tr>
+                                            </c:forEach>
+                                        </tbody>
+                                    </table>
+                                </div>
+                            </div>
+                            <div class="col-md-6">
+                                <div class="table-div table-responsive">
+                                    <table class="table table-fixed table-hover table-striped table-fixed-small">
+                                        <thead>
+                                        <tr class="tr-fixed-small">
+                                            <th>Time</th>
+                                            <th>Scorer</th>
+                                            <th>Assist</th>
+                                        </tr>
+                                        </thead>
+                                        <tbody class="tbody-scrollable-small">
+                                        <c:forEach items="${matchDetail.awayGoals}" var="awayGoal">
+                                                <tr class="tr-fixed-small">
+                                                    <td><c:out value="${awayGoal.matchTime.format(formatter)}"/></td>
+                                                    <td><c:out value="${awayGoal.scorer.firstname} ${awayGoal.scorer.surname}"/></td>
+                                                    <td><c:out value="${awayGoal.assistant.firstname} ${awayGoal.assistant.surname}"/></td>
+                                                    <sec:authorize access="hasAnyRole('ROLE_ADMIN', 'ROLE_MODERATOR')">
+                                                        <td>
+                                                            <a href="#deleteAwayGoalModal" title="Delete goal" class="btn btn-block btn-goal-delete" data-toggle="modal" role="button">
+                                                                <strong>X</strong>
+                                                            </a>
+                                                        </td>
+                                                        <div class="modal fade" id="deleteAwayGoalModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true" style="display: none;">
```

Comment:
buďto to `aria-` vyplň něčím rozumným, nebo ten attribut smaž.. slouží to jako pomoc pro čtečky pro nevidomé